### ISSUE TITLE
installation failed from pipenv.

### ISSUE THREAD

**Issue by zdyh** [polm/fugashi#1]
```
Installing dependencies from Pipfile.lock (039fd1)…
An error occurred while installing fugashi==0.1.4 --hash=sha256:0dbb394b9d21bf48f3c1772fe247da9c8fe3a53d257a8d10e23941eed86b768d! Will try again.
  🐍   ▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉ 23/23 — 00:00:46
Installing initially failed dependencies…
[pipenv.exceptions.InstallError]:   File "/Users/user/.pyenv/versions/3.7.5/envs/env-sample/lib/python3.7/site-packages/pipenv/core.py", line 1874, in do_install
[pipenv.exceptions.InstallError]:       keep_outdated=keep_outdated
[pipenv.exceptions.InstallError]:   File "/Users/user/.pyenv/versions/3.7.5/envs/env-sample/lib/python3.7/site-packages/pipenv/core.py", line 1253, in do_init
[pipenv.exceptions.InstallError]:       pypi_mirror=pypi_mirror,
[pipenv.exceptions.InstallError]:   File "/Users/user/.pyenv/versions/3.7.5/envs/env-sample/lib/python3.7/site-packages/pipenv/core.py", line 859, in do_install_dependencies
[pipenv.exceptions.InstallError]:       retry_list, procs, failed_deps_queue, requirements_dir, **install_kwargs
[pipenv.exceptions.InstallError]:   File "/Users/user/.pyenv/versions/3.7.5/envs/env-sample/lib/python3.7/site-packages/pipenv/core.py", line 763, in batch_install
[pipenv.exceptions.InstallError]:       _cleanup_procs(procs, not blocking, failed_deps_queue, retry=retry)
[pipenv.exceptions.InstallError]:   File "/Users/user/.pyenv/versions/3.7.5/envs/env-sample/lib/python3.7/site-packages/pipenv/core.py", line 681, in _cleanup_procs
[pipenv.exceptions.InstallError]:       raise exceptions.InstallError(c.dep.name, extra=err_lines)
[pipenv.exceptions.InstallError]: ['Collecting fugashi==0.1.4 (from -r /var/folders/91/7npclg6s2mn6z73kqn22fxj00000gn/T/pipenv-j3baxj38-requirements/pipenv-x686vbno-requirement.txt (line 1))', '  Using cached https://files.pythonhosted.org/packages/41/68/a9c829e26a7d5c058a482c542f16f2e8b63e12a51110d55418cf6e1dbe67/fugashi-0.1.4.tar.gz']
[pipenv.exceptions.InstallError]: ['ERROR: Command errored out with exit status 1:', '     command: /Users/user/.pyenv/versions/3.7.5/envs/env-sample/bin/python -c \'import sys, setuptools, tokenize; sys.argv[0] = \'"\'"\'/private/var/folders/91/7npclg6s2mn6z73kqn22fxj00000gn/T/pip-install-csbpfj56/fugashi/setup.py\'"\'"\'; __file__=\'"\'"\'/private/var/folders/91/7npclg6s2mn6z73kqn22fxj00000gn/T/pip-install-csbpfj56/fugashi/setup.py\'"\'"\';f=getattr(tokenize, \'"\'"\'open\'"\'"\', open)(__file__);code=f.read().replace(\'"\'"\'\\r\\n\'"\'"\', \'"\'"\'\\n\'"\'"\');f.close();exec(compile(code, __file__, \'"\'"\'exec\'"\'"\'))\' egg_info --egg-base pip-egg-info', '         cwd: /private/var/folders/91/7npclg6s2mn6z73kqn22fxj00000gn/T/pip-install-csbpfj56/fugashi/', '    Complete output (5 lines):', '    Traceback (most recent call last):', '      File "<string>", line 1, in <module>', '      File "/private/var/folders/91/7npclg6s2mn6z73kqn22fxj00000gn/T/pip-install-csbpfj56/fugashi/setup.py", line 5, in <module>', '        from Cython.Build import cythonize', "    ModuleNotFoundError: No module named 'Cython'", '    ----------------------------------------', 'ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.']
ERROR: ERROR: Package installation failed...
```

**Comment by polm** [comment]
Thanks for the report, this seems to be an issue with the way the Cython dependency is specified. 

I'll fix that, but for now it should work if you install Cython first and then install fugashi.

**Comment by polm** [comment]
This was fixed in #2. Thanks!
